Docs/source/Hardware/Boards.rst

```diff
@@ -68,23 +68,115 @@ A switching regulator (NCP1117) converts the +5V input to +3.3V. This does not n
 
     Leader board power distribution
 
-Fan Board: Manthan Upadhyaya
-=============================
+Fan Board
+=========
 
-Purpose
+Overview
+^^^^^^^^
+`GitHub Link <https://github.com/lhr-solar/BPS-FanPCB>`__
+
+`BOM Link <https://www.mouser.com/ProjectManager/ProjectDetail.aspx?AccessID=272bad62cd>`__ 
+
+Brief Description/Purpose:
     The purpose of this board is to set the speed of the fans that will be used to cool the battery 
-    pack. It recieves signals from the Master Board and sends them across a MOSFET that switches the 
-    fan on or off.
+    pack. It receives signals from the Leader Board and sends them across a :term:`MOSFET <MOSFET>` that switches 
+    the fan on or off. 
+
+Pertinent Regulations
+^^^^^^^^^^^^^^^^^^^^^
+========== ============================================== ===============================================
+Regulation Description of Regulation                      How Regulation is Met
+
+**8.4.D**  | Battery enclosures may be equipped with a    | The BPS fan board is powered by the battery 
+           | forced ventilation system. Such ventilation  | system and in the event of a battery protection 
+           | systems must pull exhaust to the exterior of | fault it's powered by the supplemental battery.
+           | the solar car and must be directly connected 
+           | to the exterior of the vehicle away from any 
+           | airstream that may reach the driver. The 
+           | ventilation system shall be powered by the 
+           | battery system. In the event of a Battery 
+           | Protection Fault, provisions should be made 
+           | to power this fan from the Supplemental 
+           | battery.
+========== ============================================== ===============================================
+
+Context
+^^^^^^^
+**Location of the Board:** With the BPS in the battery box 
+
+List of I/O and Connections: 
```

Comment:
Do you think this should be bold/italicized like some of the other headings?

Reply:
I made it bold